data/human/transport missions.txt

```diff
@@ -3304,3 +3304,199 @@ mission "Spacediving amateurs"
+
+
+mission "Mafia Extortion"
+	source
+		government "Pirate"
+	on accept
+		payment -200000
+	to offer
+		random < 2
+		credits >= 2000000
+	on offer
+		conversation
+			`A man in a tailored suit, sunglasses and shiny shoes with green hair approaches you. "Hey, that's a very nice ship you have there. Would be a shame if anything happened to it."`
+			`You notice that he is missing a finger on each hand.`
+			choice
+				`"Thank you. My name is <first> <last>. Whom do I have the pleasure of talking to?"`
+					goto nameless
+				`"Yes, I'm very happy with my ship."`
+					goto specifics
+				`"What should I be afraid of happening to my ship?"`
+					goto specifics
+				`"You sound like one of those Mafia goons."`
+					goto mafia
+				`"Are you trying to extort me?"`
+					goto mafia
+				`"Yes, it helps me transport stuff for some of the most dangerous people on this planet."`
+					goto unnerved
+				`"I don't have time to talk."`
+					goto funeral
+			label nameless
+			`Your greeting gets a grumpy reply about the value of anonymity in a place like this.`
+			choice
+				`"Sounds like you are afraid of talking to me. Why is that?"`
+					goto mafia
+				`"What should I be afraid of happenning to my ship?"`
+					goto specifics
+				`"You sound like one of those mafia goons."`
+					goto mafia
+				`"Are you trying to extort me?"`
+					goto mafia
+			label specifics
+			`The man responds, "Well, a fleet of unfriendly ships in this system might just blow your ship to pieces."`
+			choice
+				`"Ha, business as usual in this system."`
+					goto unconcerned
+				`"Are you trying to extort me?"`
+					goto mafia
+				`"And you have a way to prevent this from happening?"`
+					goto mafia
+				`"And you are the local goon that takes bribes to keep such ships docked?"`
+					goto mafia
+				`"That would be unfortunate, I run jobs for some of the most dangerous people on this planet and they wouldn't be happy if that happened."`
+					goto unnerved
+				`"I really don't have time to talk to you."`
+					goto funeral
+			label unconcerned
+			`The man responds, "Maybe you want to improve that situation?`
+				goto donation
+			label mafia
+			`The man acts as if he is shocked by your remark. "No way! I'm just a simple man walking around the spaceport and talking about life. You would be wise to listen to locals with experience, especially in dangerous places like this."`
+			label donation
+			`	"If you care about safety, then I could make a donation to the System Safety Foundation on your behalf. 200,000 credits really goes a long way around here."`
+			choice
+				`(pay 200,000 credits)`
+					#payment -200000
+					goto next_steps
```

Comment:
Did you meant for this choice to go to "next_steps"?

Reply:
No, I'll update this to go to `payup`.